src/components/GameMode.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { SyntheticEvent, useState } from "react";
+import styled from "styled-components";
+
+const GameModeWrapper = styled.div``;
+const RadioWrapper = styled.section``;
+
+interface InputProps {
+  onChange: React.ChangeEventHandler<HTMLInputElement>;
+}
+const Input = styled.input<InputProps>``;
+
+interface GameModeProps {
+  ws: WebSocket;
+}
+
+const GameMode: React.FC<GameModeProps> = ({ ws }) => {
+  const [selectedOption, setSelectedOption] = useState("cpu");
+  const [disabled, setDisabled] = useState(false);
+  const toggle = (previous: boolean) => !previous;
+
+  const startOnClickHandler = () => {
+    console.log("Start button clicked");
+    const modeObj = {
+      type: "mode",
+      mode: selectedOption,
+    };
+    ws.send(JSON.stringify(modeObj));
+    setDisabled(toggle);
```

Comment:
```suggestion
    setDisabled(false);
```
is probably simpler, and closer to what you mean (TIL: you can pass a function into the useState callback. I suppose it's similar to how setState() works on stateful components. Nice)